infrastructure/template.yaml

```diff
@@ -1837,6 +1837,61 @@ Resources:
       Threshold: 0
       TreatMissingData: missing
 
+  AuditEventExceededAgeLimitAlarm:
+    Condition: "DeployAlarms"
+    Type: AWS::CloudWatch::Alarm
+    Properties:
+      OKActions:
+        - !ImportValue platform-alarm-warning-alert-topic
+      AlarmActions:
+        - !ImportValue platform-alarm-warning-alert-topic
+      AlarmDescription: !Sub "${AuditEventResponseReceivedRule} has a queue item that is over 24 hours old in requests in the last hour. ${SupportManualURL}"
+      AlarmName: !Sub "${AWS::StackName}-${Environment}-AuditEventExceededAgeLimitAlarm"
+      MetricName: "ApproximateAgeOfOldestMessage"
+      Namespace: "AWS/Events"
+      ComparisonOperator: GreaterThanThreshold
+      Statistic: Maximum
+      DatapointsToAlarm: 1
+      EvaluationPeriods: 10
+      Period: 300
+      Threshold: 86400
+      TreatMissingData: missing
+      Dimensions:
+        - Name: RuleName
+          Value: !Select [ 1, !Split [ "|", !Ref TxMaAuditEventRule ] ]
+        - Name: EventBusName
+          Value: !Ref CheckHmrcEventBus
+
+  AuditEventDeadLetterQueueAlarm:
+    Condition: "DeployAlarms"
+    "Type": "AWS::CloudWatch::Alarm"
+    "Properties":
+      "AlarmName": !Sub "${AWS::StackName}-${Environment}-AuditEventDeadLetterQueueAlarm:"
+      "AlarmDescription": !Sub "${AuditEventResponseReceivedRule} has a request get through to the dead letter queue request in the last hour. ${SupportManualURL}"
+      "ActionsEnabled": true
+      OKActions:
+        - !ImportValue platform-alarm-warning-alert-topic
+      AlarmActions:
+        - !ImportValue platform-alarm-warning-alert-topic
+      "InsufficientDataActions": [ ]
+      "MetricName": "NumberOfMessagesReceived"
```

Comment:
Maybe we should use the `DeadLetterInvocations` metric - let me know what you think

Reply:
if we're now going to attach directly SQS Queue, `DeadLetterInvocations` could be useful, bav used `ApproximateNumberOfMessagesVisible` for theirs